metabolomics.yaml.lock

```diff
@@ -55,7 +55,17 @@ tools:
 - name: w4mclassfilter
   owner: eschen42
   revisions:
-  - 191a720488ce
+  - 1ced8b5dfa3e
```

Comment:
This file is automatically generated. Please do not edit it.

Reply:
Rather, please do not remove the old revision, that should continue to be listed. If we need to update a tool that isn't an IUC tool, then that will require a user-committed change to a lock file.

@eschen42 you can use our script to update just the w4m tools in this file:

```
python scripts/update-tool.py --owner eschen42 metabolomics.yaml
```